src/lib.rs

```diff
@@ -156,6 +156,26 @@ impl Repository {
         };
         execute_git_fn(&self.location, args, |output| output.trim().to_owned())
     }
+
+    /// Execute user defined command
+    pub fn utils<I, S>(&self, args: I) -> Result<()>
+    where
+        I: IntoIterator<Item = S>,
+        S: AsRef<OsStr>,
+    {
+        execute_git(&self.location, args)
+    }
+
+    /// Execute user defined command with fun
```

Comment:
don't think its immediately obvious what `fun` is in this context

Reply:
OK. What do you sugggest to differe to run command with and without output from console? I know this kind of naming from https://docs.rs/cmd_lib_core/0.4.0/cmd_lib_core/.

How about renaming:
- utils -> cmd
- utils_fun -> cmd_out

Reply:
Apologies for my slowness in getting back on this, yep those names lgtm :)

Reply:
Done